pkg/engine/armvariables.go

```diff
@@ -163,6 +168,12 @@ func getK8sMasterVars(cs *api.ContainerService) (map[string]interface{}, error)
 		}
 		masterVars["environmentJSON"] = environmentJSON
 		masterVars["provisionConfigsCustomCloud"] = getBase64EncodedGzippedCustomScript(kubernetesCSECustomCloud)
+
+		if cs.Properties.OrchestratorProfile.IsAzureCNI() {
```

Comment:
Add cs.Properties.IsAzureStackCloud() condition as well.

Reply:
It is already in an if block for IsAzureStackCloud() starting on line 159

Reply:
Code snippet didnt show as it was wraped.